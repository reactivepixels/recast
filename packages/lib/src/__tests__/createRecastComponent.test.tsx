import React from "react";
import { render } from "@testing-library/react";
import { MockButton } from "./mockButton.js";

describe("MockButton", () => {
  it("should render the component with the correct props", () => {
    const { container } = render(<MockButton />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
