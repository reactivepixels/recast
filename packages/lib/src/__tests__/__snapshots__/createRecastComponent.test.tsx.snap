// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MockButton should render the component with the correct props 1`] = `
<button
  class=""
/>
`;
